doc/develop/west/install.rst

```diff
@@ -48,50 +48,59 @@ APIs directly. See :ref:`west-apis` for details.
 Enabling shell completion
 *************************
 
-West currently supports shell completion in the following combinations of
-platform and shell:
+West currently supports shell completion in the following shells:
 
-* Linux: bash
-* macOS: bash
-* Windows: not available
+* bash
+* zsh
+* fish
 
 In order to enable shell completion, you will need to obtain the corresponding
-completion script and have it sourced every time you enter a new shell session.
+completion script and have it sourced.
+Using the completion scripts:
 
-To obtain the completion script you can use the ``west completion`` command::
+.. tabs::
 
-   cd /path/to/zephyr/
-   west completion bash > ~/west-completion.bash
+  .. group-tab:: bash
 
-.. note::
+    *One-time setup*:
+
+    .. code-block:: bash
+
+      source <(west completion bash)
+
+    *Permanent setup*:
+
+    .. code-block:: bash
+
+      west completion bash > ~/west-completion.bash
```

Comment:
is this really enough? the initial instructions mentioned that this file must be created *AND* sourced form .bashrc? Is this automatically done based on the name of the file? or you simply forgot it?

Reply:
You're right, it's not enough. There is multiple options to edit the line:
1. `west completion bash | sudo tee /usr/share/bash-completion/completions/west-completion.bash` 
2. `west completion bash | sudo tee /etc/bash_completion.d/west-completion.bash`
3. `west completion bash > ~/west-completion.bash; echo "source ~/west-completion.bash" >> ~/.bashrc`

IMO the 1 is the most okay-ish. But maybe without the `sudo` for both 1 and 2.
Do you have an opinion on which one we should choose?

Reply:
Updated the PR with option 3 as it's the most compatible (work even if bash-completion not installed as far as I understand).